coops-ui/src/components/StandaloneDirectorChange/CODDate.vue

```diff
@@ -0,0 +1,241 @@
+<template>
+  <v-card flat class="container">
```

Comment:
Because of a layout change I made in https://github.com/bcgov/lear/pull/523, this class should be renamed to "cod-date-container" and have the following SCSS:
```
.cod-date-container {
  padding: 1.25rem;
}
```

Reply:
Resolved in PR #556.